scripts/MRK17_Exp1_rk01.jmd

```diff
@@ -17,24 +17,27 @@ using DrWatson
 
 using MixedModels
 using CSV, DataFrames, DataFramesMeta, RCall 
-using Distributions: mean
+using Statistics: mean
 ```
 
 # Reading data
 
 We read the data preprocessed with R and saved as RDS file (see `DataPrep.Rmd` for details).
 
 ```julia
-R"dat_r = readRDS($datadir('MRK17_Exp1.rds'))";
+dat = rcopy(R"readRDS($datadir('MRK17_Exp1.rds'))");
+describe(dat)
+```
```

Comment:
I think we should be in the habit of calling `describe` on a `DataFrame` as soon as we read or import it.